README.md

```diff
@@ -110,21 +113,27 @@ Deploy the stack using the following command
 cdk deploy
 ```
 
-### Examples 
+This will provision the following:
 
-To view more examples for how you can leverage `cdk-eks-blueprint`, see the `examples` directory.
+- [x] A new Well-Architected VPC with both Public and Private subnets.
```

Comment:
Do we need a WA review before we call this out? I am concerned that if the cluster does not align with any of the WA recommendation for EKS, then a partner trying to get certification will point to this section.

Reply:
Fine by me

Reply:
We should, I am just pointing that it is a distinct activity that requires drilling into the WA spec. I know, WA is slow to deliver container lense. I am just curious about what specific guidelines are WA for EKS before we can call out we are compliant.